funnel/views/membership.py

```diff
@@ -65,6 +65,20 @@ def new_member(self):
 
         if request.method == 'POST':
             if membership_form.validate_on_submit():
+                if not membership_form.user.data.has_verified_contact_info():
+                    # users without verified contact information cannot be members
+                    return (
+                        {
+                            'status': 'error',
+                            'message': _(
+                                "User does not have any verified contact information"
```

Comment:
"This user does not have any verified contact information. If you are able to contact them, please ask them to verify their email address or phone number"

Reply:
The gender-neutral "them" is going to be a problem in translation. Will require "his or her" in the target language. It's fine for English though.